### ISSUE TITLE
[Bug]: Text overflow problems on dashboard cards on NXC 25.0.2 RC2

### ISSUE THREAD

**Issue by ilicz** [nextcloud/server#35646]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] Nextcloud Server **is** running on 64bit capable CPU, PHP and OS.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

Dashboard cards items overflow over the right edge of the cards Unread mail, Recent activity and Upcoming events.
![image](https://user-images.githubusercontent.com/98597308/206183014-5851ab5a-b1bc-4950-bb31-400e34f04157.png)

(I have not tested any more cards)

### Steps to reproduce

0. Upgrade Nextcloud instance from 24.0.7 to 25.0.2 RC3 (or any 25.0.x version)
1. Create corresponging items in each app (Event with long enough title, any unread email message...)
2. Open NXC dashboard.
3. Observe overflows.

### Expected behavior

No overflows, content truncated/wrapped correctly.

### Installation method

Community Manual installation with Archive

### Operating system

Debian/Ubuntu

### PHP engine version

PHP 8.0

### Web server

Apache (supported)

### Database engine version

MariaDB

### Is this bug present after an update or on a fresh install?

Updated to a major version (ex. 22.2.3 to 23.0.1)

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### What user-backends are you using?

- [ ] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

```shell
{
    "system": {
        "activity_expire_days": 30,
        "activity_use_cached_mountpoints": true,
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "dbtype": "mysql",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "default_language": "cs_CZ",
        "default_locale": "cs_CZ",
        "default_phone_region": "CZ",
        "filelocking.enabled": true,
        "htaccess.RewriteBase": "\/",
        "installed": true,
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "loglevel": 2,
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_sendmailmode": "smtp",
        "mail_smtpmode": "sendmail",
        "maintenance": false,
        "memcache.local": "\\OC\\Memcache\\Redis",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "mysql.utf8mb4": true,
        "overwrite.cli.url": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "timeout": 0
        },
        "secret": "***REMOVED SENSITIVE VALUE***",
        "sharing.minSearchStringLength": 2,
        "skeletondirectory": "\/srv\/data\/***REMOVED SENSITIVE VALUE***\/__skeleton",
        "templatedirectory": "\/srv\/data\/***REMOVED SENSITIVE VALUE***\/__templates",
        "theme": "",
        "trusted_domains": [
            "***REMOVED SENSITIVE VALUE***"
        ],
        "updater.release.channel": "beta",
        "version": "25.0.2.2",
        "app_install_overwrite": [
            "impersonate"
        ],
        "updater.secret": "***REMOVED SENSITIVE VALUE***"
    }
}
```

### List of activated Apps

```shell
- activity: 2.17.0
  - announcementcenter: 6.4.0
  - audioplayer: 3.3.1
  - bruteforcesettings: 2.5.0
  - calendar: 4.1.0
  - circles: 25.0.0
  - cloud_federation_api: 1.8.0
  - comments: 1.15.0
  - contacts: 5.0.1
  - contactsinteraction: 1.6.0
  - dashboard: 7.5.0
  - dav: 1.24.0
  - deck: 1.8.2
  - event_update_notification: 2.0.0
  - federatedfilesharing: 1.15.0
  - files: 1.20.1
  - files_antivirus: 4.0.1
  - files_external: 1.17.0
  - files_linkeditor: 1.1.13
  - files_mindmap: 0.0.27
  - files_pdfviewer: 2.6.0
  - files_rightclick: 1.4.0
  - files_sharing: 1.17.0
  - files_trashbin: 1.15.0
  - files_versions: 1.18.0
  - firstrunwizard: 2.14.0
  - group_everyone: 0.1.11
  - groupfolders: 13.1.0
  - guests: 2.3.0
  - impersonate: 1.11.0
  - logreader: 2.10.0
  - lookup_server_connector: 1.13.0
  - mail: 2.2.1
  - nextcloud_announcements: 1.14.0
  - notifications: 2.13.1
  - oauth2: 1.13.0
  - password_policy: 1.15.0
  - photos: 2.0.1
  - polls: 4.1.0-beta4
  - provisioning_api: 1.15.0
  - quota_warning: 1.15.0
  - ransomware_protection: 1.14.0
  - recommendations: 1.4.0
  - related_resources: 1.0.3
  - richdocuments: 7.0.1
  - richdocumentscode: 22.5.802
  - serverinfo: 1.15.0
  - settings: 1.7.0
  - sharebymail: 1.15.0
  - snappymail: 2.22.7
  - spreed: 15.0.2
  - support: 1.8.0
  - survey_client: 1.13.0
  - systemtags: 1.15.0
  - text: 3.6.0
  - theming: 2.0.1
  - theming_customcss: 1.12.0
  - twofactor_backupcodes: 1.14.0
  - twofactor_nextcloud_notification: 3.5.0
  - twofactor_totp: 7.0.0
  - updatenotification: 1.15.0
  - user_status: 1.5.0
  - viewer: 1.9.0
  - weather_status: 1.5.0
  - workflow_media_converter: 1.7.0
  - workflowengine: 2.7.0
```

### Nextcloud Signing status

```shell
No errors have been found.
```

### Nextcloud Logs

```shell
no dashboard related linex in nextcloud.log
```

### Additional info

Google Chrome: Version 108.0.5359.95 (Official Build) (64-bit)
Microsoft Edge: Version 107.0.1418.62 (Official build) (64-bit)

**Comment by ilicz** [comment]
This seems to be fixed for Polls with 25.0.2, but definitely not for Recent activities on Dashboard.
![image](https://user-images.githubusercontent.com/98597308/207035425-b2b6369c-0b1d-4c48-8af7-9f193e8e0712.png)

**Comment by dartcafe** [comment]
@ilicz This bug is still caused by polls and affects the widget 'Recent activity'.
